### ISSUE TITLE
Metaware linker can't open file

### ISSUE THREAD

**Issue by aravinds92** [foss-for-synopsys-dwc-arc-processors/toolchain#154]
I'm trying to compile some code from a bunch of C files into an executable. I wrote a makefile to compile and the link using the Metaware toolchain. However, after the compilation stage, I get the following error message whenever I try to link the object files: "[ldac] Can't open file ./aacd_hufftables.o"
I get this same error message for every .o file in the directory.
Any help on this is highly appreciated.
Aravind

**Comment by iguryanov** [comment]
Hi Aravind, 

I am pretty sure you have problems with path to object files. 
But as this is Metaware, not GNU toolchain, you should file a case against DesignWare ARC Tools / MetaWare at https://solvnet.synopsys.com. 

Regards,
Igor.

**Comment by aravinds92** [comment]
Hey Igor
Thanks for responding. That was my first thought as well. So I created a new project in the Metaware IDE and tried to build it. I got the same message as well. Considering that all of the build files are generated automatically by Eclipse, I find it hard to believe that it is a path issue.
PS: I changed the compiler in my Makefile to gcc and the GNU linker was able to locate the files in that directory.

Aravind
